app/models/html_document.rb

```diff
@@ -49,10 +49,13 @@ def parse_content
   end
 
   def tag_scrubber
-    # convert custom tags to plain 'ol divs to ensure they are not
-    # stripped out during HTML sanitization
     Loofah::Scrubber.new do |node|
+      # convert custom tags to plain 'ol divs to ensure they are not
+      # stripped out during HTML sanitization
       node.name = 'div' if /-/.match(node.name)
+
```

Comment:
Its more performant if you write this node.name =~ /-/.  https://rubocop.readthedocs.io/en/latest/cops_performance/#performanceredundantmatch

Reply:
Will update.

Reply:
Note: Once we are on ruby 2.4+, the preferred method will be `.match?`:
https://rubocop.readthedocs.io/en/latest/cops_performance/#performanceregexpmatch

Reply:
Yes in 2.4  its because the methods avoid creating a MatchData object or saving backref.  It also recommends using .match? instead of .match. But right now we don't have match? so the more performant way to do it is node.name =~ /-/.